src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -259,6 +270,7 @@ function MetadataSidebarRedesign({ api, elementId, fileId, onError, isFeatureEna
 
 export { MetadataSidebarRedesign as MetadataSidebarRedesignComponent };
 export default flow([
+    withRouter,
```

Comment:
can we remove this and use `useHistory` within the `useMetadataSidebarFilteredTemplates` to get access to the history?